awx/main/management/commands/create_preload_data.py

```diff
@@ -10,7 +10,7 @@
 class Command(BaseCommand):
     """Create preloaded data, intended for new installs
     """
-    help = 'Creates a preload tower data iff there is none.'
+    help = 'Creates a preload tower data if there is none.'
```

Comment:
I'll bet this isn't a typo 😉

https://en.wikipedia.org/wiki/If_and_only_if#Origin_of_iff_and_pronunciation

Reply:
My suspicions are confirmed @matburt!

https://github.com/ansible/awx/commit/8c4e4f4c0d282889eeda057bd510cfdfaad13def

Reply:
aha! good point.  thanks for pointing that out.